ci/docker/runtime_functions.sh

```diff
@@ -1404,6 +1429,21 @@ test_artifact_repository() {
     popd
 }
 
+integration_test() {
+    # install gradle
+    add-apt-repository ppa:cwchien/gradle
+    apt-get update -y
+    apt-get install gradle -y
```

Comment:
remove this line

Reply:
Remove everything apt related here. That belongs into a Dockerfile.

Reply:
Removed all apt-get related codes